starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -3,16 +3,19 @@
 import com.starter.fullstack.api.Inventory;
 import com.starter.fullstack.dao.InventoryDAO;
 import java.util.List;
+import javax.validation.Valid;
 import org.springframework.util.Assert;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 /**
  * Inventory Controller.
  */
 @RestController
-@RequestMapping("/inventory")
+@RequestMapping("/inventories")
```

Comment:
Great job on this ticket! Your code is exactly what we look for. I just have one small question: why did you change "/inventory" to "/inventories"?